docs/.custom_wordlist.txt

```diff
@@ -45,3 +45,25 @@ VMs
 VPN
 websockets
 yaml
+STSStack
+sshuttle
+vpn
+behavior
+HH
+YYYY
+ss
```

Comment:
Please remove these and put the date format in docs/how-to/purge-data-on-shadow-table.rst into preformatted text.

```suggestion
```